vividus-tests/src/main/resources/story/integration/ElementStepsTests.story

```diff
@@ -0,0 +1,179 @@
+Description: Integration tests for ElementSteps class.
+
+Meta:
+    @group vividus-plugin-web-app
+
+Lifecycle:
+Examples:
+|URL                                                                                    |URL2                                                                                                |
+|https://mdn.mozillademos.org/en-US/docs/Web/HTML/Element/textarea$samples/Basic_example|https://mdn.mozillademos.org/en-US/docs/Web/HTML/Element/input/radio$samples/Defining_a_radio_group?|
```

Comment:
`https://mdn.mozillademos.org/en-US/docs/Web` - maybe could be moved to some global-property?